pwiz_tools/Skyline/TestPerf/SmallMolLibrariesTutorialTest.cs

```diff
@@ -0,0 +1,431 @@
+            doc = WaitForDocumentChange(doc);
+
+            //   The results must now be reimported with the newly applied IMS settings.
+            //   �	From the Edit menu, click Manage Results.
+            //   �	Click the Re-import button. �*� Should appear to the left of F_A_018.
+            //   �	Select M_A_001 in the Manage Results view.
+            //   �	Click the Re-import button. �*� Should appear to the left of M_A_001.
+            //   �	Click the OK button.
+            doc = SkylineWindow.Document;
+            RunDlg<ManageResultsDlg>(SkylineWindow.ManageResults, manageDlg =>
+            {
+                manageDlg.SelectedChromatograms = SkylineWindow.Document.Settings.MeasuredResults.Chromatograms.ToArray();
+                manageDlg.ReimportResults();
+                manageDlg.OkDialog();
+            });
+            //   This should start the re-import and cause Skyline to show the Importing Results progress form.
+            doc = WaitForDocumentChangeLoaded(doc);
+
+            //   �	Click any other lipid in the Target list and re-select PE(16:1_18:3) to update the chromatograms.
+            FindNode("PC(16:0_18:1)");
+            WaitForGraphs();
+            FindNode("PE(16:1_18:3)");
+            WaitForGraphs();
+            //   To explore the filtered data, perform the following:
+            //   �	Click on the apex of the blue precursor chromatogram to show the Full-Scan graph.
+            ClickChromatogram(F_A_018, 14.807, 152.0E3, PaneKey.PRECURSORS);
+            //   �	Click the Zoom to Selection button.
+            RunUI(() => SkylineWindow.GraphFullScan.ZoomToSelection(true));
+            //   The Full-Scan graph should now look something like this:
+            PauseForScreenShot("Full scan graph with IM filtering", 21);
+
+            if (IsPauseForCoverShot)
+            {
+                RestoreCoverViewOnScreen(false);
+                PauseForCoverShot();
+                return;
+            }
+
+            // Note that if you were interested in lipids that are not present in the current spectral library, you can add to it manually or using LipidCreator. To access the LipidCreator plugin, do the following:
+            //   �	From the Tools menu, click Tool Store.
+            var configureToolsDlg = ShowDialog<ConfigureToolsDlg>(SkylineWindow.ShowConfigureToolsDlg);
+            //   �	Select LipidCreator.
+            var pick = ShowDialog<ToolStoreDlg>(configureToolsDlg.AddFromWeb);
+            var hasLipidCreator = false;
+            RunUI(() =>
+            {
+                const string toolName = "LipidCreator";
+                hasLipidCreator = pick.HasTool(toolName);
+                if (hasLipidCreator)
+                {
+                    pick.SelectTool(toolName);
+                }
+            });
+            PauseForScreenShot(hasLipidCreator ? "LipidCreator in tool store" : "LipidCreator is not installed, do that first then take screen shot", 22);
```

Comment:
I don't think this has anything to do with LipidCreator installation on the current computer. This would be missing if the ToolStore did not have LipidCreator in it. So, I think you need to clarify this message. I also think you want to put this check inside "IsPauseForScreenShots" to avoid any problems that might occur with this form trying to contact the website during TeamCity and nightly testing.

Reply:
>> avoid any problems that might occur with this form trying to contact the website during TeamCity and nightly testing
yeah, that's what I was thinking about - I'll make sure this is actually dealing with it properly